ansible/roles/nginx/templates/etc/nginx/sites-available/default.conf.j2

```diff
@@ -633,13 +633,13 @@ server {
         ssl_certificate           {{ nginx_tpl_ssl_certificate }};
         ssl_certificate_key       {{ nginx_tpl_ssl_certificate_key }};
         ssl_protocols             {{ (item.tls_protocols | d(nginx_default_tls_protocols)) | join(" ") }};
-        ssl_prefer_server_ciphers on;
+        ssl_prefer_server_ciphers off;
```

Comment:
I agree that we should stick to preferring the server ciphers instead of the client specified ones. This can prevent downgrade attacks and shouldn't be an issue even if that's not relevant anymore with TLSv1.3.

Reply:
After reading about changes in TLSv1.3, I guess client-selected ciphers make sens when it is used. But if TLSv1.2 is also available, I would stick to server ciphers. Therefore this needs to be enabled/disabled conditionally, similarly to how DHE params are dealt with. Don't forget about the instance in the `ssl.conf` snippet as well.

Reply:
Per drybjeds suggestions, I'll disable server-preferred ciphers only if TLSv1.3 is the only protocol in use.